---
permalink: /research/
title: "Research"
author_profile: true
redirect_from: 
  - /research/
  - /research.html
---

“Everything flows and nothing stands still.” -Heraclitus (quoted by Plato in Cratylus, 402a)\
\
For organisms, the natural environments they live in are not static; neither are their genomes. My research seeks to enhance our understanding of how environmental changes shape genetic variation and impact genome evolution. The connections among environment, phenotype, and genotype are mediated by molecular traits such as gene expression and epigenetic modification. Accordingly, I focus on these molecular traits to decipher the underlying mechanisms of environmental effects on genetic variation. There are three fundamental processes that shape genetic variation: natural selection, mutation, and recombination. My previous work explored how environmental change affects natural selection that shapes genetic variation and gene expression variation. My current work focuses on the modifications at chromatin (epigenetic modification), which change chromatin proteins without changing the DNA and can impact how cells read the underlying genetic code. Despite evidence showing their functional importance and environmental sensitivity, little is known about their impacts on the three processes and genome evolution.\
![research flowchat](/images/research-flowchat.png)

## Research Topic 1: Gene expression and chromatin modification mediate selection
**Past work**: Environmentally fluctuating selection has been theoretically predicted to maintain genetic variation within populations. In my PhD, I tested this hypothesis using Drosophila experimental populations that evolved on constant or temporally and spatially fluctuating diets. For loci under ecological differential selection, fluctuating diets maintained higher diversity than the constant diets, providing strong support for the theory ([Huang et al. 2014](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004527)). Because gene expression is a molecular trait that connects the genotype and fitness, studying how selection acts on gene expression can provide mechanistic insight into how certain genotypes achieve higher fitness. I studied the transcriptomes of the populations reared on either selective diet and identified diet-sensitive transcripts, which are presumably ecologically relevant traits. Interestingly, I found that most environmentally induced expression changes were opposed by selection ([Huang and Agrawal. 2016](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006336)). My postdoctoral work further showed that the pattern observed in my experimental evolution project is applicable to natural populations. I studied the transcriptomes of three closely related pairs of natural fly populations, reflecting three separate instances of adaptation to cold habitats ([Huang et al. 2021](https://academic.oup.com/genetics/article/218/3/iyab077/6275754)), and I found that selected changes in gene expression tended to oppose the naive plastic responses ([Huang et al. 2022](https://academic.oup.com/gbe/article/14/4/evac050/6563504)). These results suggest that expression plasticity induced by a novel environment may be maladaptive.\
\
**Current work**: I am focusing on another molecular trait that can mediate selection – chromatin modification, given its environmental sensitivity. Chromatin state can be influenced by the presence of transposable elements (TEs), a major type of structural mutation that can self-replicate and potentially disrupt gene function. Hosts have evolved to repress TE transposition by depositing repressive chromatin marks (e.g., H3K9me2/3) around TEs. In several model organisms, such repressive chromatin marks can spread around the TE and further exacerbate its harmfulness. Using multiple related Drosophila species, I found evidence that stronger TE-mediated local enrichment of repressive marks leads to stronger selection against TEs. Investigating these observations in a comparative framework, I found that enrichment of repressive chromatin marks is negatively correlated with the TE copy numbers within clades, suggesting that the magnitude of TE silencing can explain the variation of TE abundance between closely related species ([Huang, Shukla, & Lee. 2022](https://elifesciences.org/articles/81567)). I further used forward simulations to explore how a new allele of chromatin modifying enzymes that results in the enrichment of repressive marks around TEs may increase in frequency within populations. Quite surprisingly, I found that the conditions for the modifier to spread are restricted, unless when incorporating the suppressive effects of TE-mediated silencing on recombination ([Huang & Lee. 2024](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2023.2775)).\
![K9 screenshot](/images/K9_igv_screenshot.png)

## Research Topic 2: Chromatin modification and recombination landscape
Meiotic recombination is not only a key component of meiosis but also an important parameter shaping patterns of genetic variation. Despite its importance, detailed linkage maps have only been generated for few model organisms due to the difficulty of genotyping a large number of individuals. Nevertheless, these studies show that recombination rates can vary between species locally (mega- or kilobase-level) and globally (chromosome-level). Because repressive chromatin marks can reduce accessibility of recombination machinery, I hypothesize that TE-mediated repressive chromatin modification reduces local recombination rates. To test that hypothesis, I studied how polymorphic TEs in euchromatin regions impact local crossover events in two inbred lines. I developed a novel approach using long-read PacBio sequencing to efficiently identify crossover events among pooled recombinant individuals, with a low false positive rate (6.5%), randomly distributed false-negative events, and a high resolution of recombination breakpoints (<1 kb). This approach mitigates the need for sequencing individual flies and can readily be applied to organisms that produce many progeny or to gametes in bulk. Consistent with the hypothesis, I found that the presence of TEs, particularly those that lead to local enrichment of repressive marks, reduces local crossover rates ([Huang et al. 2024](https://www.biorxiv.org/content/10.1101/2024.09.17.613564v1.abstract)).

## Research Topic 3: Role of chromatin modification in mutation rate and types
Mutation is the ultimate source of genetic variation. My goal is to understand the role of chromatin modification in shaping the mutation process, especially given that chromatin modification and mutation are sensitive to environmental conditions. It is widely debated how the chromatin state influences the rates of point mutations. In addition, it is currently unknown how natural variation in chromatin state influences TE transposition rate, a critical knowledge gap for studying the variation of TE abundance and TE silencing ([Huang & Lee. 2024](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2023.2775)).\
\
**Ongoing work**: I am experimentally testing the impacts of TE silencing on mutations. I focus on a gene called *Su(var)3-9*, which has a well-established chromatin-modifying function. To manipulate the spreading of repressive chromatin marks, we inserted an extra *Su(var)3-9* copy with varying expression levels into a common genetic background. We performed mutation accumulation experiments using ~50 lines per genotype under full-sib mating to minimize selection, and then whole-genome sequencing. This dataset would allow me to study how TE transposition and mutation rate change with different levels of repressive chromatin marks.
